packages/elements-dev-portal/package.json

```diff
@@ -47,7 +47,7 @@
   },
   "dependencies": {
     "@fortawesome/free-solid-svg-icons": "^5.10.2",
-    "@stoplight/elements-core": "^7.0.0",
+    "@stoplight/elements-core": "~7.0.0",
```

Comment:
Can we enforce >= 7.0.1 here? According to our best knowledge 7.0.0 does **not** work, so would you make this `~7.0.1` instead? Same for `elements`.